src/components/card.js

```diff
@@ -16,7 +16,11 @@ class Card extends Component {
     event.preventDefault();
     const guess = this._guess.value;
     this._guess.value = "";
-    this.props.guess(guess);
+    if(guess === '') {
+      this.handleHintClick(event)
```

Comment:
I missed it during pairing, but now that I look at it on GitHub I notice that we missed the trailing semicolon here. In this case it's unambiguous, and we don't yet have a linter hooked up, but would you mind adding one?

Reply:
Done